### PR TITLE
Resolving merge conflict

### PR DESCRIPTION
**Pull request by demewozshiferaw**
## Changes
Added a step to take when merge conflict happen 

## Purpose
Resolve with merge conflict 

## Testing Steps
link not 404

## Learning
N/A

Closes #6

### CODE REVIEW

**Review by fedluk70 — COMMENTED** [review]
Looks cool

File: README.md [hunk 1]
```diff
@@ -18,6 +18,9 @@ https://docs.github.com/en/authentication/troubleshooting-ssh/error-permission-d
 
 *All combined notes, links and documents are listed below:
 
+ds-6-useful-links
+practicing
```
Comment (michaelachrisco) [inline README.md]: Lets remove the branch names and "test" or "practicing" messages within the documentation.

File: README.md [hunk 2]
```diff
@@ -39,28 +42,49 @@ https://docs.github.com/en/authentication/troubleshooting-ssh/error-permission-d
 
 
 
+ development
```
Comment (michaelachrisco) [inline README.md]: Same here, lets remove this.

File: README.md [hunk 3]
```diff
@@ -39,28 +42,49 @@ https://docs.github.com/en/authentication/troubleshooting-ssh/error-permission-d
...
 ## Git Commands
 
+### https://youtu.be/kqtD5dpn9C8
```
Comment (michaelachrisco) [inline README.md]: Why is this link turned into a header? My suggestion would be to remove the header. Like so: 
```suggestion
https://youtu.be/kqtD5dpn9C8
```

File: README.md [hunk 4]
```diff
@@ -39,28 +42,49 @@ https://docs.github.com/en/authentication/troubleshooting-ssh/error-permission-d
...
+
 ### 1. Docs
 - [Article](https://dzone.com/articles/top-20-git-commands-with-examples)
 
 - [Cheat-sheet](https://phoenixnap.com/kb/wp-content/uploads/2021/11/git-commands-cheat-sheet-by-pnap-v2.pdf)
 
+
+### Resolving merge conflict 
+
+ HEAD
```
Comment (michaelachrisco) [inline README.md:62]: Lets remove the HEAD line. Please refer to the merge conflict talk that we had if you are confused on how to do this.

File: README.md [hunk 5]
```diff
@@ -39,28 +42,49 @@ https://docs.github.com/en/authentication/troubleshooting-ssh/error-permission-d
...
+- [resolving-a-merge-conflict-using-the-command-line](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/addressing-merge-conflicts/resolving-a-merge-conflict-using-the-command-line)
```
Comment (michaelachrisco) [inline README.md:63]: Nice link!

File: README.md [hunk 6]
```diff
@@ -39,28 +42,49 @@ https://docs.github.com/en/authentication/troubleshooting-ssh/error-permission-d
...
    - [merge-conflict-using-the-command-line](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/addressing-merge-conflicts/resolving-a-merge-conflict-using-the-command-line) 
 
 
+ds-6-useful-links
+## [Learn Infrastructure as Code](https://youtu.be/7xngnjfIlK4)
+
+## [learn python](https://youtu.be/kqtD5dpn9C8
+
+eefe0f61787d3a0bdda3c9aebfd7427223eadd41
+
```
Comment (michaelachrisco) [inline README.md:80]: Im thinking you may want to rework this change.

File: README.md [hunk 7]
```diff
@@ -39,28 +42,49 @@ https://docs.github.com/en/authentication/troubleshooting-ssh/error-permission-d
...
 ## useful terraform cheat sheet 
 - [terraform-cheat-sheet2](https://k21academy.com/terraform-iac/terraform-cheat-sheet/)
+development
```
Comment (michaelachrisco) [inline README.md:83]: Lets remove the branch names like stated above.

**Review by michaelachrisco — CHANGES_REQUESTED** [review]
Thanks for adding so many new resources! Thanks for the contribution!

In order to resolve the merge conflict, my suggestion would be to close this PR at this time and remake the branch from scratch. Sometimes its just easier to do that then go over the many many merge conflicts. Let me know if you need help with that.

**Comment by michaelachrisco** [comment]
Closing all old Prs over 1 month old. Feel free to re-submit!

**Pull request closed without merging**